### PR TITLE
null value removed & scala style applied 

### PR DESCRIPTION
**Pull request by ranraj**
User Group null value removed
Most of the scala style standards fixed except Main.scala file and few others like ( method exceed a maximum length)

### CODE REVIEW

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala [hunk 1]
```diff
@@ -1,112 +1,137 @@
...
+import scala.collection.immutable.HashMap
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+/**
+  * Created by sivag on 6/10/16.
+  */
+object AppSettingsNeo4jWrapper {
+
+  val labels: HashMap[String, String] =
```
Comment (eklavya) [inline src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Fix formatting

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala [hunk 2]
```diff
@@ -1,112 +1,137 @@
...
+    else{
+      updateOrDeleteSettings(settingsToUpdate, labels("HGS").toString, "UPDATE")
+    }
+
+  }
+
+  def deleteSetting(settingsToDelete: Map[String, String], settingsType: String): Future[ExecutionStatus] = {
+    if (settingsType.equalsIgnoreCase("AUTH_SETTINGS")){
+      updateOrDeleteSettings(settingsToDelete, labels("HAS").toString, "DELETE")
+    }
```
Comment (eklavya) [inline src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Fix formatting

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala [hunk 3]
```diff
@@ -1,112 +1,137 @@
...
+            case Some(rootNode) => val relationNode = getRelationNodeByName(rootNode, relationName)
+              relationNode match {
+                case Some(dbnode) =>
+                  val todelete = if (updateOrDelete.equalsIgnoreCase("DELETE")) true else false
+                  settings.foreach {
+                    case (k, v) =>
+                      if (todelete){
+                        dbnode.removeProperty(k)
+                      }
+                      else{
```
Comment (eklavya) [inline src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Fix formatting

**Comment by ranraj** [comment]
WIP

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala (hunk 2 above)
Comment (ranraj) [reply src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Done with refactoring

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala (hunk 3 above)
Comment (ranraj) [reply src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Formating done with refactoring

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala (hunk 1 above)
Comment (ranraj) [reply src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Done

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala (hunk 3 above)
Comment (ranraj) [reply src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Format issue fixed with the below code 

```
 case (k, v) => if (toDelete) dbNode.removeProperty(k) else dbNode.setProperty(k, v.toString)
```

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala (hunk 2 above)
Comment (ranraj) [reply src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Refactored with following code,

```
private def label(settingsType: String) =
    if (settingsType.equalsIgnoreCase("AUTH_SETTINGS")) labels("HAS") else labels("HGS")
```

File: src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala (hunk 1 above)
Comment (ranraj) [reply src/main/scala/com/imaginea/activegrid/core/models/AppSettingsNeo4jWrapper.scala]: Formated as per the below reference 
http://docs.scala-lang.org/style/indentation.html

**Pull request merged**